pandas_gbq/gbq.py

```diff
@@ -815,6 +816,13 @@ def to_gbq(dataframe, destination_table, project_id, chunksize=10000,
         Service account private key in JSON format. Can be file path
         or string contents. This is useful for remote server
         authentication (eg. jupyter iPython notebook on remote host)
+    table_schema : list of dicts
+    .. versionadded:: 0.2.0
```

Comment:
the versionadded should go after the expl (with a blank line before ), otherwise doesn't render.